### PR TITLE
feat(releases): Secondary event type selector for event count

### PR DESCRIPTION
**Pull request by Zylphrex**
Add a secondary selector when y axis is Event Count to allow selection of the
type of event. Also update to the new this vs other releases style chart.

### CODE REVIEW

File: src/sentry/static/sentry/app/views/releases/detail/overview/chart/utils.tsx [hunk 1]
```diff
@@ -40,62 +41,83 @@ export function getReleaseEventView(
...
+      ) {
+        const eventTypeFilter = eventType === 'all' ? '' : `event.type:${eventType}`;
+        return EventView.fromSavedQuery({
+          ...baseQuery,
+          query: stringifyQueryObject(
+            new QueryResults([releaseFilter, eventTypeFilter].filter(Boolean))
+          ),
+        });
+      } else {
+        // TODO(tonyx): Delete this else once the feature flags are removed
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/releases/detail/overview/chart/utils.tsx:103]: Won't this always depend on the organization having `performance-view` or `discover-basic`?

Comment (Zylphrex) [reply src/sentry/static/sentry/app/views/releases/detail/overview/chart/utils.tsx:103]: This else block exists to make sure the existing behaviour does not change until we remove the `release-performance-views` feature flag. Without this check, the existing `All Events` y-axis option will immediately begin using the new query.

Comment (Zylphrex) [reply src/sentry/static/sentry/app/views/releases/detail/overview/chart/utils.tsx:103]: I think removing that `performance-view` feature flag check above would make the intent here clearer. When the selected Y-Axis is `YAxis.Events`, we have 2 possible views.

1. The new view with this release vs other releases which is available with the `release-performance-views` feature flag
2. The existing view of the non-transaction events that we are replacing.

**Review by matejminar — APPROVED** [review]
Would it make sense to remove `eventType` from the URL once we switch Y-Axis to something other than Event Count? So that they are as clean as possible?

Other than that, looking great 🚀

**Pull request merged**